edxval/tests/test_api.py

```diff
@@ -1170,10 +1183,13 @@ def test_transcript_export(self):
         self.assert_xml_equal(exported_metadata['xml'], expected_xml)
 
         # Verify transcript file is created.
-        self.assertEqual(sorted(transcript_files.values()), sorted(self.file_system.listdir(constants.EXPORT_IMPORT_STATIC_DIR)))
+        self.assertEqual(
+            sorted(transcript_files.values()),
+            sorted(self.file_system.listdir(constants.EXPORT_IMPORT_STATIC_DIR))
+        )
 
         # Also verify the content of created transcript file.
-        for language_code in transcript_files.keys():
+        for language_code in transcript_files.keys():  # pylint: disable=consider-iterating-dictionary
```

Comment:
That warning was actually good advice, this should probably just be `for language_code in transcript_files:`.